src/components/addComment.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from "react"
+import { addCommentToArticle } from "../utils";
+
+const AddComment = (props) => {
+
+    const [input, setInput] = useState("")
+    const [date, setDate] = useState(new Date().toISOString());
+    const {article_id} = props;
+    const [errorMessage, setErrorMessage] = useState("")
+
+    const updateInput = (event) => {
+        setInput(event.target.value)
+    } 
+
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+
+        props.setComments((currComments) => {
+            
+            if (input.length === 0) {
+                return[...currComments]
+            }
+
+            setDate(new Date().toISOString())
```

Comment:
This is an interesting idea, but using setDate here would only update this after a re-render, so you wouldnt never have access to the correct date for this new comment. This also doesnt actually need to be saved in state as it is a variable that only needs to exist when we create a new comment.